v-next/hardhat/templates/02-mocha-ethers/ignition/modules/Apollo.ts

```diff
@@ -0,0 +1,9 @@
+import { buildModule } from "@ignored/hardhat-vnext-ignition/modules";
```

Comment:
I got an eslint error here requiring me to add `@ignored/hardhat-vnext-ignition` as a dev dependency.
We should think through whether we are keeping the same policy for importing `buildModule` as we did in HH2.

Reply:
That eslint rule is right. It should be installed.